portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro

```diff
@@ -86,4 +86,55 @@ definition {
 		return "${articleId}";
 	}
 
+	macro _createStructuredContentDraft {
+		var portalURL = JSONCompany.getDefaultPortalURL();
+
+		var siteId = JSONGroupAPI._getGroupIdByName(
+			groupName = "Guest",
+			site = "true");
+
+		var curl = '''
+			${portalURL}/o/headless-admin-content/v1.0/sites/${siteId}/structured-contents/draft \
+			-u test@liferay.com:test \
+			-H Content-Type: application/json
+            -d
```

Comment:
similar to first test, I recommend listing out the required parameters needed for each macro with `Variables.assertDefined(...)`